apps/api/src/app/change/usecases/get-changes/get-changes.usecase.ts

```diff
@@ -168,4 +173,28 @@ export class GetChanges {
       templateName: item.name,
     };
   }
+
+  private async getTemplateDataForLayout(
+    entityId: string,
+    environmentId: string
+  ): Promise<IViewEntity | Record<string, unknown>> {
+    let item = await this.layoutRepository.findOne({
+      _environmentId: environmentId,
+      _id: entityId,
+    });
```

Comment:
I have seen that this module has no dependencies beyond the SharedModule and AuthModule. But I consider we shouldn't call the Layout repository here and take use of the Layout module use cases.
Here, we would need the `GetLayoutUseCase`. And then we would need to implement the use case to find a deleted layout.